lib/paths-cache.js

```diff
@@ -399,18 +400,31 @@ export default class PathsCache extends EventEmitter {
         .replace(/\./g, '\\.')
         .replace(/\*/g, '.*')
     );
-    const ignorePattern = '\'.*\\(' + ignorePatternsForFind.join('\\|') + '\\).*\'';
-
-    const cmd = 'find'
-    const args = [
-      '-L',
-      directoryPath + '/',
-      '-type', 'f',
-      '-not', '-regex', ignorePattern,
-    ];
-
-    const stdout = await spawn(cmd, args)
-    const files = _.compact(stdout.toString().split('\n'));
+
+    let files
+    if (process.platform === 'win32') {
+      const { stdout } = await execFile(
+        'dir',
+        [
+          '/A', // Show all files
+          '/S', // Include all subfolders
+          '/B', // Bare format (no heading, file sizes or summary)
+          directoryPath
+        ], {shell: true, maxBuffer: 10000*10000})
+        const ignorePatternRegexp =  new RegExp(ignorePatternsForFind.join("|"))
+        files = _.compact(stdout.split('\n')).filter(f => !f.match(ignorePatternRegexp))
```

Comment:
The results are split by `\r\n`

It works after this change.

```suggestion
        files = _.compact(stdout.split(/\r?\n/)).filter(f => !f.match(ignorePatternRegexp))
```